### PR TITLE
Fix South Carolina spidering

### PR DESCRIPTION
**Pull request by CloCkWeRX**
Fix https://github.com/alltheplaces/alltheplaces/issues/9659

### CODE REVIEW

File: locations/spiders/south_carolina_us.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+        for i in data:
+            try:
+                item = DictParser.parse(i)
+                item["lon"] = -abs(float(i["Longitude"]))
+                item["name"] = i["Description"]
+
+                category_mapping = cat[int(i["CategoryId"]) - 1]
+
+                if category_mapping == "Libary":
```
Comment (matkoniecz) [inline locations/spiders/south_carolina_us.py:33]: is it really Libary?

File: locations/spiders/south_carolina.py [hunk 2]
```diff
@@ -26,21 +24,21 @@ def parse(self, response):
...
-
-                yield Feature(**properties)
+                item = DictParser.parse(i)
+                item["lon"] = -abs(float(i["Longitude"]))
+                item["country"] = "US"
+                item["name"] = i["Description"]
+
+                category_mapping = cat[int(i["CategoryId"]) - 1]
+
+                if category_mapping == "Libary":
```
Comment (CloCkWeRX) [inline locations/spiders/south_carolina.py:34]:
```suggestion
                if category_mapping == "Library":
```

File: locations/spiders/south_carolina_us.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import json
+
+import scrapy
+
+from locations.categories import Categories
```
Comment (CloCkWeRX) [inline locations/spiders/south_carolina_us.py:5]:
```suggestion
from locations.categories import Categories, apply_category
```

File: locations/spiders/south_carolina_us.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import json
+
+import scrapy
+
+from locations.categories import apply_category
```
Comment (CloCkWeRX) [inline locations/spiders/south_carolina_us.py:5]:
```suggestion
from locations.categories import Categories, apply_category
```

File: locations/spiders/south_carolina_us.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        for i in data:
+            try:
+                item = DictParser.parse(i)
+                item["lon"] = -abs(float(i["Longitude"]))
+                item["name"] = i["Description"]
+
+                category_mapping = cat[int(i["CategoryId"]) - 1]
+
+                if category_mapping == "Libary":
+                    apply_category(Category.LIBRARY, item)
```
Comment (CloCkWeRX) [inline locations/spiders/south_carolina_us.py:37]:
```suggestion
                    apply_category(Categories.LIBRARY, item)
```

**Pull request merged**